vmdb/app/models/orchestration_stack_openstack_infra.rb

```diff
@@ -0,0 +1,13 @@
+class OrchestrationStackOpenstackInfra < OrchestrationStack
+  def raw_update_stack(options)
+    ext_management_system.with_provider_connection(:service => "CloudFormation") do |connection|
```

Comment:
Yes, I am implementing the correct body in the next pull request oriented solely on scaling of Openstack. This should not be here. :-)